dlna/dms/cds_windows.go

```diff
@@ -0,0 +1,36 @@
+//+build windows
+
+package dms
+
+import (
+	"os"
+	"unicode/utf16"
+
+	"golang.org/x/sys/windows"
+)
+
+const hiddenAttributes = windows.FILE_ATTRIBUTE_HIDDEN | windows.FILE_ATTRIBUTE_SYSTEM
+
+func isHiddenPath(path string) (hidden bool, err error) {
+	attrs, err := windows.GetFileAttributes(toWindowsPath(path))
```

Comment:
Do you still want to treat dot files, for "unix-on-windows" style use of the filesystem or does this handle it?

Reply:
Nope. This only handles the Windows "hidden" and "system" attributes, which is Windows' idiomatic way to hide files.

Reply:
However, I think this will not check for parent status.